### PR TITLE
Disable Travis (SOFTWARE-4252)

### PR DESCRIPTION
**Pull request by brianhlin**
Due to inactivity, the cron actions were disabled on this repo. We need to sort out how to prevent that since this triggers the weekly builds of all our other containers

### CODE REVIEW

**Comment by brianhlin** [comment]
Oh, if I remembered having an open PR I wouldn't have squashed and force pushed so much...

Anyway, the latest stuff combines the EL7/EL8 branches, since GHA scheduled crons can only run on the default branch, which was a concern originally mentioned in the Jira ticket.

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+       YUM_PKG_NAME="yum-plugin-priorities" || \
+       YUM_PKG_NAME="yum-utils" && \
+    yum update -y && \
     yum -y install http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
                    epel-release \
-                   yum-plugin-priorities && \
+                   $YUM_PKG_NAME && \
     yum -y install supervisor cronie && \
+    [[ $IMAGE_BASE_TAG == centos7 ]] || yum-config-manager --enable PowerTools && \
+    [[ $IMAGE_BASE_TAG == centos7 ]] || yum-config-manager --enable osg-testing && \
```
Comment (matyasselmeci) [inline Dockerfile:20]:
```suggestion
    if [[ $IMAGE_BASE_TAG != centos7 ]]; then \
        yum-config-manager --enable PowerTools && \
        yum-config-manager --enable osg-testing; \
    fi; \
```

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+
+# "ARG IMAGE_BASE_TAG" needs to be here again because the previous instance has gone out of scope.
+ARG IMAGE_BASE_TAG=centos7
 
 LABEL maintainer OSG Software <help@opensciencegrid.org>
 
-RUN yum update -y && \
+RUN [[ $IMAGE_BASE_TAG == centos7 ]] && \
+       YUM_PKG_NAME="yum-plugin-priorities" || \
+       YUM_PKG_NAME="yum-utils" && \
```
Comment (matyasselmeci) [inline Dockerfile:13]:
```suggestion
RUN \
    if  [[ $IMAGE_BASE_TAG == centos7 ]]; then \
       YUM_PKG_NAME="yum-plugin-priorities"; \
    else \
       YUM_PKG_NAME="yum-utils"; \
    fi; \
```

**Review by matyasselmeci — APPROVED** [review]
A few style changes to make the Dockerfile less confusing but otherwise it looks good; pre-approving.

File: Dockerfile (hunk 1 above)
Comment (brianhlin) [reply Dockerfile:20]: Shouldn't that be

```
fi && \
```

to retain the same logic?

**Comment by matyasselmeci** [comment]
Sure, that would work too.

**Comment by brianhlin** [comment]
Could you update the suggestions? Then I'll get them and this PR merged

**Review by matyasselmeci — APPROVED** [review]

**Comment by matyasselmeci** [comment]
Done.

**Pull request merged**